localization/ekf_localizer/src/ekf_module.cpp

```diff
@@ -282,15 +283,37 @@ bool EKFModule::measurement_update_pose(
   return true;
 }
 
-geometry_msgs::msg::PoseWithCovarianceStamped EKFModule::compensate_pose_with_z_delay(
+geometry_msgs::msg::PoseWithCovarianceStamped EKFModule::compensate_rph_with_delay(
   const PoseWithCovariance & pose, const double delay_time)
 {
+  tf2::Quaternion delta_orientation;
+  if (last_angular_velocity_.length() > 0.0) {
+    delta_orientation.setRotation(
+      last_angular_velocity_.normalized(), last_angular_velocity_.length() * delay_time);
+  } else {
+    delta_orientation.setValue(0.0, 0.0, 0.0, 1.0);
+  }
+
+  tf2::Quaternion prew_orientation = tf2::Quaternion(
+    pose.pose.pose.orientation.x, pose.pose.pose.orientation.y, pose.pose.pose.orientation.z,
+    pose.pose.pose.orientation.w);
+
+  tf2::Quaternion curr_orientation;
+  curr_orientation = prew_orientation * delta_orientation;
```

Comment:
Is this a typo of `prev`?

Reply:
Sorry for missing this. I fixed with this commit --> https://github.com/autowarefoundation/autoware.universe/pull/8095/commits/33989271a435a9856c46c36fbb6918433a9384cc